Adyen/Model/Checkout/Installments2.cs

```diff
@@ -0,0 +1,140 @@
+#region Licence
+
+// 
+//                        ######
+//                        ######
+//  ############    ####( ######  #####. ######  ############   ############
+//  #############  #####( ######  #####. ######  #############  #############
+//         ######  #####( ######  #####. ######  #####  ######  #####  ######
+//  ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
+//  This file is open source and available under the MIT license.
+//  See the LICENSE file for more info.
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Adyen.Model.Checkout
+{
+    /// <summary>
+    /// Installments2
+    /// </summary>
+    [DataContract]
+    public partial class Installments2 : IEquatable<Installments2>, IValidatableObject
```

Comment:
bit ugly naming, this is because there are 2 classes named `Installments` in the specs, can you please report it to Checkout?

Reply:
Addressed to checkout